### PR TITLE
More evident link to gene coverage in Var page

### PR DESCRIPTION
**Pull request by northwestwitch**
Addressing [this comment](https://github.com/Clinical-Genomics/scout/issues/3581#issuecomment-1241693658)

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: scout/server/blueprints/variant/templates/variant/components.html [hunk 1]
```diff
@@ -82,8 +82,8 @@
             <div class="d-flex flex-wrap">
               {% for gene in variant.genes %}
                 <span class="ms-5">
-                  <a target="_blank" href="{{ url_for('report.gene', gene_id=gene.hgnc_id, sample_id=variant.samples|map(attribute='sample_id')|list) }}">
-                    {{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}
+                  <a class="btn btn-sm btn-secondary" target="_blank" href="{{ url_for('report.gene', gene_id=gene.hgnc_id, sample_id=variant.samples|map(attribute='sample_id')|list) }}">
+                    gene coverage{{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/components.html:86]:
```suggestion
                    Gene coverage {{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}
```

**Review by dnil — APPROVED** [review]

**Comment by northwestwitch** [comment]
I'll try it real quick stage before merging!

**Comment by northwestwitch** [comment]
> I'll try it real quick stage before merging!

image
 
Ugly! I'll fix

File: scout/server/blueprints/variant/templates/variant/components.html (hunk 1 above)
Comment (dnil) [reply scout/server/blueprints/variant/templates/variant/components.html]: @northwestwitch did you miss this? 😉

File: scout/server/blueprints/variant/templates/variant/components.html [hunk 2]
```diff
@@ -82,8 +82,8 @@
             <div class="d-flex flex-wrap">
               {% for gene in variant.genes %}
                 <span class="ms-5">
-                  <a target="_blank" href="{{ url_for('report.gene', gene_id=gene.hgnc_id, sample_id=variant.samples|map(attribute='sample_id')|list) }}">
-                    {{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}
+                  <a class="btn btn-sm btn-secondary" target="_blank" href="{{ url_for('report.gene', gene_id=gene.hgnc_id, sample_id=variant.samples|map(attribute='sample_id')|list) }}">
```
[the remark this answers is not in the document]
Comment (dnil) [reply scout/server/blueprints/variant/templates/variant/components.html:85]: Quite ok! I liked the space style better, but this is perfectly fine!

[the remark this answers is not in the document]
Comment (northwestwitch) [reply scout/server/blueprints/variant/templates/variant/components.html:85]: Too late, that's what happens when you approve before testing ;)

**Pull request merged**